__tests__/secret.test.js

```diff
@@ -110,4 +110,26 @@ describe('Tozny identity client', () => {
     expect(result[0].data.secretValue).toBe('secret-value')
     expect(result[0].meta.plain.secretType).toBe('Credential')
   })
+  it('can create a secret and update', async () => {
+    const secret = {
+      secretType: 'Credential',
+      secretName: `test-secret-updatetest35558800`,
+      secretValue: 'secret-value',
+      description: 'this is a description',
+    }
+    await ops.createSecret(realmConfig, identity, secret)
+    await new Promise(r => setTimeout(r, 1000))
+    const query = await ops.getSecrets(realmConfig, identity, 10)
+    let secrets = await query.next()
+    await ops.updateSecret(realmConfig, identity, secret)
+    await new Promise(r => setTimeout(r, 10000))
+    const query2 = await ops.getSecrets(realmConfig, identity, 100)
+    const newSecrets = await query2.next()
+    let length1 = secrets.length
+    let length2 = newSecrets.length
+    expect(length1 + 1).toBe(length2) // this shows we created a new secret
+    expect(secrets[length1 - 1].meta.plain.version).toBe('latest') // this shows that the last item is latest
```

Comment:
Your comment doesn’t seem to line up with the code here, secrets is the original list of secrets after you created the first one (think of you how you can rename these variables to be more meaningful and clear

 Secrets could be `singleSecretVersionListResponse`
NewSecrets could be `multipleSecretVersionsListResponse`

)